Automattic/wp-calypso
# Tests: Move Features List tests to single test runner

This pull request fixes https://github.com/Automattic/wp-calypso/issues/3942 by updating the unit test suite of the `Features List` component to use the `client` test runner.

```
$ make test

  FeaturesList
    initialize
      ✓ should populate the list of features
      ✓ should set attributes properly

  2 passing (181ms)

```

#### Testing instructions
 
1. Run `git checkout update/features-list-tests`
2. Execute `make test` and check that all tests for the module pass
